pybabylonjs/args.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2022 TileDB Inc.
+# Licensed under the MIT License.
+"""Functions to format and check the data and keyword arguments for each data source and visualization mode."""
+
+import os
+import sys
+from .data import *
+
+POINT_CLOUD_KWARG_DEFAULTS = {
+    "inspector": False,
+    "width": 800,
+    "height": 600,
+    "z_scale": 1,
+    "wheel_precision": -1,
+    "point_size": 1,
+    "bbox": {"X": [0, 1], "Y": [0, 1], "Z": [0, 1]},
+    "time_offset": 0,
+    "classes": {"numbers": [], "names": []},
+    "mbtoken": "none",
+    "mbstyle": "streets-v11",
+    "crs": "EPSG:2994",
+    "topo_offset": 0,
+    "gltf_data": "none",
```

Comment:
`gltf_data` should be in the schema, but a default should be `None`

Reply:
added